fmriprep/workflows/base.py

```diff
@@ -217,60 +219,118 @@ def basic_wf(subject_data, settings, name='fMRI_prep'):
     # Preprocessing of T1w (includes registration to MNI)
     t1w_pre = t1w_preprocessing(settings=settings)
 
+    workflow.connect([
+        (bidssrc, t1w_pre, [('t1w', 'inputnode.t1w'),
+                            ('t2w', 'inputnode.t2w')])
+    ])
+
+    if settings['freesurfer']:
+        workflow.connect([
+            (inputnode, t1w_pre, [('subjects_dir', 'inputnode.subjects_dir')]),
+            ])
+
+    for bold_file in subject_data['func']:
+        name = os.path.split(bold_file)[-1].replace(".", "_")
+        metadata = layout.get_metadata(bold_file)
+        bold_pre = bold_preprocessing(name=name, metadata=metadata,
+                                      settings=settings)
+        bold_pre.get_node('inputnode').inputs.epi = bold_file
+
+        workflow.connect([
+            (bidssrc, bold_pre, [('t1w', 'inputnode.t1w')]),
+            (t1w_pre, bold_pre,
+             [('outputnode.bias_corrected_t1', 'inputnode.bias_corrected_t1'),
+              ('outputnode.t1_brain', 'inputnode.t1_brain'),
+              ('outputnode.t1_mask', 'inputnode.t1_mask'),
+              ('outputnode.t1_seg', 'inputnode.t1_seg'),
+              ('outputnode.t1_tpms', 'inputnode.t1_tpms'),
+              ('outputnode.t1_2_mni_forward_transform', 'inputnode.t1_2_mni_forward_transform')])
+        ])
+
+        if settings['freesurfer']:
+            workflow.connect([
+                (inputnode, bold_pre,
+                 [('subjects_dir', 'inputnode.subjects_dir')]),
+                (t1w_pre, bold_pre,
+                 [('outputnode.subject_id', 'inputnode.subject_id'),
+                  ('outputnode.fs_2_t1_transform', 'inputnode.fs_2_t1_transform')]),
+            ])
+
+    return workflow
+
+
+def bold_preprocessing(name, metadata, settings):
```

Comment:
Can we deprecate the `fmriprep.workflows.epi` module in favor of `fmriprep.workflows.bold` module which would contain this `bold_preprocessing` workflow?